src/main/common/utils.h

```diff
@@ -33,8 +33,7 @@
 #define CONCAT3(_1,_2,_3)  CONCAT(CONCAT(_1, _2), _3)
 #define CONCAT4(_1,_2,_3,_4)  CONCAT(CONCAT3(_1, _2, _3), _4)
 
-#define STR_HELPER(x) #x
-#define STR(x) STR_HELPER(x)
+#define STR(x) #x
```

Comment:
This won't work if you pass `#define`d macro as argument. Level of indirection is necessary

```
#define TEST string
STR(TEST); // expands to "TEST", not "string"